src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,127 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String LONG_LOGIN = "roberto";
+    private static final String SHORT_LOGIN = "bob";
+    private static final String GOOD_LOGIN = "morris";
+    private static final String LOGIN_WITH_SPACE = "gary  ";
+    private static final String EMPTY_LOGIN = "";
+    private static final int VALID_AGE = 22;
+    private static final int ENOUGH_AGE = 18;
+    private static final int NOT_ENOUGH_AGE = 12;
+    private static final int NEGATIVE_AGE = -10;
+    private static final String LONG_PASSWORD = "robinson134";
+    private static final String SHORT_PASSWORD = "984";
+    private static final String GOOD_PASSWORD = "123456";
+    private static final String PASSWORD_WITH_SPACE = "robi   ";
+    private static final String EMPTY_PASSWORD = "";
```

Comment:
Maybe it`s better not to add so many static final fields. Just my opinion

Reply:
In your test classes that is OK if you use literals directly instead of constants. But, if you already have created a lot of constants, you can move them to a separate class, e.g. `RegistrationServiceTestUtils` or `RegistrationServiceTestConstants`.